test-network/scripts/deployCC.sh

```diff
@@ -33,54 +33,31 @@ CC_SRC_LANGUAGE=$(echo "$CC_SRC_LANGUAGE" | tr [:upper:] [:lower:])
 
 FABRIC_CFG_PATH=$PWD/../config/
 
-# User has not provided a path, therefore the CC_NAME must
-# be the short name of a known chaincode sample
-if [ "$CC_SRC_PATH" = "NA" ]; then
-  infoln "Determining the path to the chaincode"
-  # first see which chaincode we have. This will be based on the
-  # short name of the known chaincode sample
-  if [ "$CC_NAME" = "basic" ]; then
-    println $'\e[0;32m'asset-transfer-basic$'\e[0m' chaincode
-    CC_SRC_PATH="../asset-transfer-basic"
-  elif [ "$CC_NAME" = "events" ]; then
-    println $'\e[0;32m'asset-transfer-events$'\e[0m' chaincode
-    CC_SRC_PATH="../asset-transfer-events"
-  elif [ "$CC_NAME" = "secured" ]; then
-    println $'\e[0;32m'asset-transfer-secured-agreeement$'\e[0m' chaincode
-    CC_SRC_PATH="../asset-transfer-secured-agreement"
-  elif [ "$CC_NAME" = "ledger" ]; then
-    println $'\e[0;32m'asset-transfer-ledger-agreeement$'\e[0m' chaincode
-    CC_SRC_PATH="../asset-transfer-ledger-queries"
-  elif [ "$CC_NAME" = "private" ]; then
-    println $'\e[0;32m'asset-transfer-private-data$'\e[0m' chaincode
-    CC_SRC_PATH="../asset-transfer-private-data"
-  elif [ "$CC_NAME" = "sbe" ]; then
-    println $'\e[0;32m'asset-transfer-sbe$'\e[0m' chaincode
-    CC_SRC_PATH="../asset-transfer-sbe"
-  else
-    fatalln "The chaincode name ${CC_NAME} is not supported by this script. Supported chaincode names are: basic, events, ledger, private, sbe, secured"
-  fi
+#User has not provided a name
+if [ -z "$CC_NAME" ]; then
```

Comment:
When I issued a command without any parameters, the script was only printing: ` fatalln "Path to chaincode does not exist. Please provide different path"` without the call example. I would make these series of If statements into a series of If then statements so you can have more control over what is printed.

Reply:
I'm not as familiar with shell scripts, could you please elaborate on your suggestion?

Reply:
Sorry, meant if else. Was playing around and got engrossed. Try this:
```
#User has not provided a name
if [ "$CC_NAME" = "NA" ]; then
  fatalln "No chaincode name was provided. Valid call example: ./network.sh deployCC -ccn basic -ccp ../asset-transfer-basic/chaincode-go -ccl go"

# User has not provided a path
elif [ "$CC_SRC_PATH" = "NA" ]; then
  fatalln "No chaincode path was provided. Valid call example: ./network.sh deployCC -ccn basic -ccp ../asset-transfer-basic/chaincode-go -ccl go"

# User has not provided a language
elif [ "$CC_SRC_LANGUAGE" = "NA" ]; then
  fatalln "No chaincode language was provided. Valid call example: ./network.sh deployCC deployCC -ccn basic -ccp ../asset-transfer-basic/chaincode-go -ccl go"

## Make sure that the path to the chaincode exists
elif [ ! -d "$CC_SRC_PATH" ]; then
  fatalln "Path to chaincode does not exist. Please provide different path"
fi

# do some language specific preparation to the chaincode before packaging

CC_SRC_LANGUAGE=$(echo "$CC_SRC_LANGUAGE" | tr [:upper:] [:lower:])
```
Moving the last line down does not shift "NA" to "na"